x-pack/legacy/plugins/ml/kibana.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "kibanaVersion": "kibana",
   "configPath": ["ml"],
+  "requiredPlugins": ["licensing"],
   "server": true,
   "ui": true
 }
```

Comment:
This file doesn't do anything in legacy. I would remove it completely.

Reply:
Removed in 609be5c4c218d1feed4b249158c438ce7c30f99d